feat(PoS): add known issues for Node Setup docs (#1638)
maticnetwork/matic-docs: docs/maintain/validate/kb/known-issues.md

If in any case, none of these solutions work for you, please contact the Polygon Support team immediately.

### Log: Error validating checkpoint module=checkpoint startBlock

If your node throws up these logs, please check the following:

> - Ensure whether the **Bor service is in sync or not**. Command to check that:

Reviewer: ```suggestion
- Ensure that the **Bor node is in sync**. To check, run the following command:
```

    ```bash
    bor attach .bor/data/bor.ipc
    eth.syncing
    ```

>     If the output is **false**, then the node is in sync.

Reviewer: ```suggestion
    If the output is **false**, then the Bor node is in sync.
```

> - Another possible cause for this issue is that **your node might be on the wrong fork**. To check that, find the current block on your node:

Reviewer: ```suggestion
- Another possible cause of this issue is that **your node might be on the wrong fork**. To check, run the following command to find the current block on your node:
```

    ```bash
    bor attach .bor/data/bor.ipc
    eth_blockNumber
    ```

>     Find the Block Hash for the Block Number:

Reviewer: ```suggestion
    Find the associated block hash for the block number:
```

    ```
    bor attach .bor/data/bor.ipc
    eth.getBlockByNumber("<Block Number>").hash
    ```

>     Now, search for the same block number on Polygonscan.

Reviewer: ```suggestion
    Now, you can inspect the block number to identify if you are running on the right fork. One way to do this is to search for the block number on an explorer like [PolygonScan](https://polygonscan.com/).
```

    <img src={useBaseUrl("img/knowledge-base/block_number.png")}/>

>     If the hashes from your node and Polygonscan matches, then the node is on the right fork.

Reviewer: ```suggestion
    If the hashes match, then the node is on the right fork.
```

### Log: Error dialing seed

> If this log throws up, please check whether your Heimdall is configured with the right seeds as listed on the [node setup docs](/docs/operate/full-node-deployment.md#full-node-setup).

Reviewer: ```suggestion
Please check whether your Heimdall node is configured with the latest seeds as listed on the [node setup documents](/docs/operate/full-node-deployment.md#full-node-setup).
```

> If so and still this error shows up, please clear the `addrbook.json` by following the below steps:

Reviewer: ```suggestion
If you're still encountering the error after either updating to the latest seeds or confirming that you are using the right seeds, you may need to clear the `addrbook.json` file. To do this, follow the steps below.
```

> 1. Open the `config.toml` file in the terminal: ```vi /var/lib/heimdall/config/config.toml```

Reviewer: ```suggestion
1. Open the `config.toml` file in your terminal: ```vi /var/lib/heimdall/config/config.toml```
```

2. Stop `heimdalld` service: ```sudo service heimdalld stop```

3. Clear your `addrbook`

    ```
    sudo service heimdalld stop
    cp /var/lib/heimdall/config/addrbook.json /var/lib/heimdall/config/addrbook.json.bkp
    rm /var/lib/heimdall/config/addrbook.json
    ```

4. Increase `max_num_inbound_peers` and `max_num_outbound_peers` in `/var/lib/heimdall/config/config.toml`:

    ```
    max_num_inbound_peers = 300
    max_num_outbound_peers = 100
    ```

5. Start `heimdalld` service: ```sudo service heimdalld start```

### Log: Demoting invalidated transaction 

> This log is not an error. It is basically a process in **txpool** which re-arranges the transactions and removes some of them (as per specific conditions).

Reviewer: ```suggestion
This log is not an error. It is a process in **txpool** which rearranges the transactions and removes some of them (as per specific conditions).
```
